dist/obs-server.spec

```diff
@@ -504,7 +561,6 @@ touch /srv/www/obs/api/last_deploy || true
 /usr/sbin/rcobssrcserver
 /usr/sbin/rcobswarden
 /usr/sbin/rcobsdodup
-/usr/sbin/rcobsdeltastore
```

Comment:
It would be nice if we would have `rc` compat symlinks for people stuck on sysvinit in their head. See ` ls -la /usr/sbin/rc*` on a SUSE system near your location :grinning:

Reply:
yes, and this does not belong to api at all, it is running usually on the source server. Please keep it.

Reply:
Keeping `/usr/sbin/rcobsdeltastore`.